packages/builder/src/steps/invoke.ts

```diff
@@ -327,6 +327,14 @@ const invokeSpec = {
     }
 
     return [
+      [
+        cfg.target?.map((t) => getContractFromPath(ctx, t)?.address), // to
+        cfg.func, // func
+        cfg.args?.map((v) => JSON.stringify(v)), // args
```

Comment:
Very small speed improv, but
```suggestion
        cfg.args ? JSON.stringify(cfg.args) : null, // args
```